2025-10-31 10:02:36,361 : INFO : root : Loading config.
2025-10-31 10:02:36,365 : INFO : YamlMultifileConfigLoader : Machine config file #1: config.yaml
2025-10-31 10:02:36,365 : INFO : ConfigProcessor : Loading config from cache: C:\Users\antho\AppData\Local\Temp\fdca14d0c9d658ab8200d57120800af5.mpf_cache
2025-10-31 10:02:36,507 : INFO : ConfigProcessor : Loading config from cache: C:\Users\antho\AppData\Local\Temp\5dec96b1547c4adf5bdd3d1715e74bc0.mpf_cache
2025-10-31 10:02:36,508 : INFO : ConfigProcessor : Loading config from cache: C:\Users\antho\AppData\Local\Temp\202a5840610e95051f9f473dfa660cc0.mpf_cache
2025-10-31 10:02:36,509 : INFO : ConfigProcessor : Loading config from cache: C:\Users\antho\AppData\Local\Temp\5dec96b1547c4adf5bdd3d1715e74bc0.mpf_cache
2025-10-31 10:02:36,509 : INFO : ConfigProcessor : Loading config from cache: C:\Users\antho\AppData\Local\Temp\2cd13a2d3f375e00ab01d6d09a85d8e7.mpf_cache
2025-10-31 10:02:36,510 : WARNING : ConfigProcessor : Config file in cache changed: C:\Users\antho\nightpin\modes\escape_graveyard\config\escape_graveyard.yaml
2025-10-31 10:02:36,510 : INFO : ConfigProcessor : Loading config from file C:\Users\antho\nightpin\modes\escape_graveyard\config\escape_graveyard.yaml.
2025-10-31 10:02:36,528 : INFO : ConfigProcessor : Loading config: C:\Users\antho\nightpin\modes\escape_graveyard\config\escape_graveyard.yaml
2025-10-31 10:02:36,529 : INFO : ConfigProcessor : Config file cache created: C:\Users\antho\AppData\Local\Temp\9220b44b97b91972524500b81035d790.mpf_cache
2025-10-31 10:02:36,530 : INFO : ConfigProcessor : Loading config from cache: C:\Users\antho\AppData\Local\Temp\31ee881584fa6fce665a672a4f3f633a.mpf_cache
2025-10-31 10:02:36,531 : INFO : ConfigProcessor : Loading config from cache: C:\Users\antho\AppData\Local\Temp\5e841763ec73085bbf92358fe6afda21.mpf_cache
2025-10-31 10:02:36,532 : INFO : ConfigProcessor : Loading config from cache: C:\Users\antho\AppData\Local\Temp\7f73d12aa9beccac1cbd8c265ccda72e.mpf_cache
2025-10-31 10:02:36,533 : INFO : ConfigProcessor : Loading config from cache: C:\Users\antho\AppData\Local\Temp\a03ebaa9d6d5676c418deb142cb34911.mpf_cache
2025-10-31 10:02:36,533 : INFO : ConfigProcessor : Loading config from cache: C:\Users\antho\AppData\Local\Temp\b17152efa55a24f0424602723f8d3921.mpf_cache
2025-10-31 10:02:36,534 : INFO : ConfigProcessor : Loading config from cache: C:\Users\antho\AppData\Local\Temp\e87fc0e7d2f8a851b5fa3f604e0d0c6e.mpf_cache
2025-10-31 10:02:36,534 : INFO : ConfigProcessor : Loading config from cache: C:\Users\antho\AppData\Local\Temp\196ac76b6cab3a9880804c826cfbcb5c.mpf_cache
2025-10-31 10:02:36,535 : INFO : ConfigProcessor : Loading config from cache: C:\Users\antho\AppData\Local\Temp\55437eb303afaa980597ff60f339a871.mpf_cache
2025-10-31 10:02:36,546 : INFO : ConfigProcessor : Loading config from cache: C:\Users\antho\AppData\Local\Temp\fddc1006228d3387e6d1e46051a3638d.mpf_cache
2025-10-31 10:02:36,557 : INFO : ConfigProcessor : Loading config from cache: C:\Users\antho\AppData\Local\Temp\834292dd7853e12a66732dc89828c828.mpf_cache
2025-10-31 10:02:36,558 : INFO : ConfigProcessor : Loading config from cache: C:\Users\antho\AppData\Local\Temp\46f8fa3e41dfeeb8232af2f03a13b6c4.mpf_cache
2025-10-31 10:02:36,559 : INFO : ConfigProcessor : Loading config from cache: C:\Users\antho\AppData\Local\Temp\c67e9c175da6c060302945929b27b878.mpf_cache
2025-10-31 10:02:36,560 : INFO : Machine : Mission Pinball Framework Core Engine v0.80.0.dev11
2025-10-31 10:02:36,560 : INFO : Machine : Command line arguments: {'no_load_cache': False, 'create_config_cache': True, 'bcp': False, 'configfile': ['config.yaml'], 'force_assets_load': False, 'platform_integration_test': False, 'jsonlogging': False, 'logfile': 'logs\\2025-10-31-10-02-36-mpf-LAPTOP-MPGP6R69.log', 'pause': False, 'production': False, 'text_ui': False, 'loglevel': 15, 'consoleloglevel': 20, 'force_platform': 'virtual', 'syslog_address': None, 'full_logfile_path': 'C:\\Users\\antho\\nightpin\\logs\\2025-10-31-10-02-36-mpf-LAPTOP-MPGP6R69.log'}
2025-10-31 10:02:36,560 : INFO : Machine : MPF path: C:\Users\antho\mpf\Lib\site-packages\mpf
2025-10-31 10:02:36,560 : INFO : Machine : Machine path: C:\Users\antho\nightpin
2025-10-31 10:02:36,560 : INFO : Machine : Platform: win32
2025-10-31 10:02:36,560 : INFO : Machine : Python executable location: C:\Users\antho\mpf\Scripts\python.exe
2025-10-31 10:02:36,560 : INFO : Machine : Python version: 3.12.7 (64-bit)
2025-10-31 10:02:36,563 : INFO : Machine : Initializing MPF...
2025-10-31 10:02:36,676 : INFO : EventManager : Event: ======'machine_var_credits_string'====== Args={'value': 'FREE PLAY', 'prev_value': None, 'change': True}
2025-10-31 10:02:36,709 : INFO : EventManager : Event: ======'machine_var_master_volume'====== Args={'value': 0.5, 'prev_value': None, 'change': True}
2025-10-31 10:02:36,709 : INFO : EventManager : Event: ======'machine_var_player1_score'====== Args={'value': 2200, 'prev_value': None, 'change': True}
2025-10-31 10:02:36,709 : INFO : EventManager : Event: ======'machine_var_player2_score'====== Args={'value': 1200, 'prev_value': None, 'change': True}
2025-10-31 10:02:36,709 : INFO : EventManager : Event: ======'machine_var_mpf_version'====== Args={'value': 'Mission Pinball Framework v0.80.0.dev11', 'prev_value': None, 'change': True}
2025-10-31 10:02:36,709 : INFO : EventManager : Event: ======'machine_var_mpf_extended_version'====== Args={'value': 'Mission Pinball Framework v0.80.0.dev11, Config version:6, Show version: 6, BCP version:1.1', 'prev_value': None, 'change': True}
2025-10-31 10:02:36,710 : INFO : EventManager : Event: ======'machine_var_python_version'====== Args={'value': '3.12.7', 'prev_value': None, 'change': True}
2025-10-31 10:02:36,852 : INFO : EventManager : Event: ======'machine_var_platform'====== Args={'value': 'Windows-11-10.0.26100-SP0', 'prev_value': None, 'change': True}
2025-10-31 10:02:36,852 : INFO : EventManager : Event: ======'machine_var_platform_system'====== Args={'value': 'Windows', 'prev_value': None, 'change': True}
2025-10-31 10:02:36,852 : INFO : EventManager : Event: ======'machine_var_platform_release'====== Args={'value': '11', 'prev_value': None, 'change': True}
2025-10-31 10:02:36,852 : INFO : EventManager : Event: ======'machine_var_platform_version'====== Args={'value': '10.0.26100', 'prev_value': None, 'change': True}
2025-10-31 10:02:36,852 : INFO : EventManager : Event: ======'machine_var_platform_machine'====== Args={'value': 'AMD64', 'prev_value': None, 'change': True}
2025-10-31 10:02:36,852 : INFO : EventManager : Event: ======'machine_var_log_file_path'====== Args={'value': 'C:\\Users\\antho\\nightpin\\logs\\2025-10-31-10-02-36-mpf-LAPTOP-MPGP6R69.log', 'prev_value': None, 'change': True}
2025-10-31 10:02:36,853 : INFO : EventManager : Event: ======'init_phase_1'====== Args={}
2025-10-31 10:02:36,896 : INFO : EventManager : Event: ======'drop_target_bank_right_bank_up'====== Args={}
2025-10-31 10:02:36,896 : INFO : EventManager : Event: ======'drop_target_bank_left_bank_up'====== Args={}
2025-10-31 10:02:36,901 : INFO : EventManager : Event: ======'init_phase_2'====== Args={}
2025-10-31 10:02:36,903 : ERROR : asyncio : Exception in callback EventManager._queue_task_done(<Task finishe...ype: 'dict'")>)
handle: <Handle EventManager._queue_task_done(<Task finishe...ype: 'dict'")>)>
Traceback (most recent call last):
  File "C:\Users\antho\AppData\Local\Programs\Python\Python312\Lib\asyncio\events.py", line 88, in _run
    self._context.run(self._callback, *self._args)
  File "C:\Users\antho\mpf\Lib\site-packages\mpf\core\events.py", line 795, in _queue_task_done
    future.result()
  File "C:\Users\antho\mpf\Lib\site-packages\mpf\core\events.py", line 708, in _run_handlers_sequential
    handler.callback(queue=queue, **merged_kwargs)
  File "C:\Users\antho\mpf\Lib\site-packages\mpf\core\mode_controller.py", line 97, in initialize_modes
    item.method(config=mode.config[item.config_section],
  File "C:\Users\antho\mpf\Lib\site-packages\mpf\core\config_player.py", line 150, in process_mode_config
    config = self.validate_config(config)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\antho\mpf\Lib\site-packages\mpf\core\config_player.py", line 113, in validate_config
    validated_config[event] = self.validate_config_entry(settings, event)
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\antho\mpf\Lib\site-packages\mpf\config_players\event_player.py", line 75, in validate_config_entry
    config = super().validate_config_entry(settings, name)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\antho\mpf\Lib\site-packages\mpf\config_players\flat_config_player.py", line 17, in validate_config_entry
    config = self._parse_config(settings, name)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\antho\mpf\Lib\site-packages\mpf\core\config_player.py", line 138, in _parse_config
    config = self.get_list_config(config)   # pylint: disable-msg=assignment-from-no-return
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\antho\mpf\Lib\site-packages\mpf\config_players\event_player.py", line 114, in get_list_config
    result[event] = {}
    ~~~~~~^^^^^^^
TypeError: unhashable type: 'dict'
2025-10-31 10:02:36,906 : INFO : Machine : Starting the main run loop with active modes: []
2025-10-31 10:02:36,906 : WARNING : Machine : Attract mode is not active, game will not be playable. Please check your attract mode configuration.
2025-10-31 10:02:36,907 : INFO : Machine : Shutting down...
2025-10-31 10:02:36,907 : INFO : EventManager : Event: ======'shutdown'====== Args={}
2025-10-31 10:02:37,919 : ERROR : Machine : Runtime Exception
Traceback (most recent call last):
  File "C:\Users\antho\mpf\Lib\site-packages\mpf\core\machine.py", line 810, in _run_loop
    raise self._exception['exception']
  File "C:\Users\antho\AppData\Local\Programs\Python\Python312\Lib\asyncio\events.py", line 88, in _run
    self._context.run(self._callback, *self._args)
  File "C:\Users\antho\mpf\Lib\site-packages\mpf\core\events.py", line 795, in _queue_task_done
    future.result()
  File "C:\Users\antho\mpf\Lib\site-packages\mpf\core\events.py", line 708, in _run_handlers_sequential
    handler.callback(queue=queue, **merged_kwargs)
  File "C:\Users\antho\mpf\Lib\site-packages\mpf\core\mode_controller.py", line 97, in initialize_modes
    item.method(config=mode.config[item.config_section],
  File "C:\Users\antho\mpf\Lib\site-packages\mpf\core\config_player.py", line 150, in process_mode_config
    config = self.validate_config(config)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\antho\mpf\Lib\site-packages\mpf\core\config_player.py", line 113, in validate_config
    validated_config[event] = self.validate_config_entry(settings, event)
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\antho\mpf\Lib\site-packages\mpf\config_players\event_player.py", line 75, in validate_config_entry
    config = super().validate_config_entry(settings, name)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\antho\mpf\Lib\site-packages\mpf\config_players\flat_config_player.py", line 17, in validate_config_entry
    config = self._parse_config(settings, name)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\antho\mpf\Lib\site-packages\mpf\core\config_player.py", line 138, in _parse_config
    config = self.get_list_config(config)   # pylint: disable-msg=assignment-from-no-return
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\antho\mpf\Lib\site-packages\mpf\config_players\event_player.py", line 114, in get_list_config
    result[event] = {}
    ~~~~~~^^^^^^^
TypeError: unhashable type: 'dict'
2025-10-31 10:02:37,921 : INFO : root : MPF run loop ended.
